daal4py/sklearn/linear_model/logistic_path.py

```diff
@@ -74,6 +84,7 @@ def __logistic_regression_path(
     max_squared_sum=None,
     sample_weight=None,
     l1_ratio=None,
+    n_threads=1,
```

Comment:
Am I right that multithreads have been used? Why it was changed?

Reply:
this is the setting for openmp parallelization. It was introduced in scikit-learn 1.1 and is always equal to 1 so far.